ddpui/api/user_org_api.py

```
@@ -137,16 +139,64 @@
+        raise HttpError(400, f"Logout failed: {str(e)}")
+
+
+@user_org_router.post("/token/refresh", auth=None)
+def post_token_refresh(request, payload: TokenRefreshPayload):
+    """Refreshes the JWT token using the refresh token"""
+    serializer = CustomTokenRefreshSerializer(data=payload.dict())
+    try:
+        serializer.is_valid(raise_exception=True)
+    except TokenError:
+        # If the refresh token is blacklisted, SimpleJWT will raise TokenError
+        raise HttpError(401, "Refresh token is invalid or blacklisted")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)